portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -39,18 +40,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     UserService userService = UserServiceFactory.getUserService();
 
-    final String USERNAME_FIELD = request.getParameter("username");
-    final String ID_FIELD = userService.getCurrentUser().getUserId();
+    final String usernameField = request.getParameter(USERNAME_KEY);
+    final String idField = userService.getCurrentUser().getUserId();
```

Comment:
As in the other file, I'd say no need to create these local variables.. you could just call these methods inline where you're using the results below.

(And if you did want to keep these local variables, I would probably remove the word "Field" from them since the names "username" and "id" describe exactly what is in the variables.)

Reply:
Added